src/net/sf/mpxj/primavera/PrimaveraDatabaseReader.java

```diff
@@ -264,8 +267,7 @@ private void processTasks() throws SQLException
    {
       List<Row> wbs = getRows("select * from " + m_schema + "projwbs where proj_id=? and delete_date is null order by parent_wbs_id,seq_num", m_projectID);
       List<Row> tasks = getRows("select * from " + m_schema + "task where proj_id=? and delete_date is null", m_projectID);
-      List<Row> costs = getRows("select * from " + m_schema + "projcost where proj_id=? and delete_date is null", m_projectID);
```

Comment:
XER files don't seem to actually contain the PROJCOST table (view?) so I had to build up the cost information myself instead of reading from PROJCOST in order to also support XER file exports.